test/OpenTelemetry.AotCompatibility.TestApp/OpenTelemetry.AotCompatibility.TestApp.csproj

```diff
@@ -14,10 +14,10 @@
       .github\workflows\ci.yml so that it runs for the project being added.
```

Comment:
What's about ci.yml? Shouldn't it be updated?

Reply:
This library was already covered by the changes in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/commit/397e9f8a2eba6fa2c72afac2d394a8698e4114cf.